requirements/test.txt

```diff
@@ -9,3 +9,4 @@ pydocstyle>=1.0.0
 virtualenv<14.0.0;python_version=='3.2'
 coverage<4.0.0;python_version=='3.2'
 beautifulsoup4>=4.1.0
+pyserial>=3.0.1
```

Comment:
Travis Error: Double requirement given: pyserial>=3.0.1 (from -r /home/travis/build/smarthomeNG/smarthome/requirements/test.txt (line 12)) (already in pyserial>=3.0.1 (from -r /home/travis/build/smarthomeNG/smarthome/requirements/all.txt (line 8)), name='pyserial')